src/hubmap_translator.py

```diff
@@ -82,6 +82,8 @@ def __init__(self, indices, app_client_id, app_client_secret, token):
         self.token = token
         self.request_headers = self.create_request_headers_for_auth(token)
         self.entity_api_url = self.indices[self.DEFAULT_INDEX_WITHOUT_PREFIX]['document_source_endpoint'].strip('/')
+        UUID_API_URL = 'https://uuid-api.dev.hubmapconsortium.org' #@TODO-un-hard-code, figure out what stuff on above line does/means
```

Comment:
We should add a new config for this uuid-api URL in https://github.com/hubmapconsortium/search-api/blob/karlburke/files-api/src/instance/search-config.yaml.example and the actual location depends on how this uuid-api URL gets called. 

https://github.com/hubmapconsortium/search-api/blob/karlburke/files-api/src/hubmap_translator.py#L872 uses the uuid-api URL, but I can't find how this method is being used.

Reply:
The method using these variables was not being used, per comment on your observation below. Removed in branch file.